GoogleForCreators/web-stories-wp
# Workspace: Hide Search Input based on feature flag

## Summary

This hides the Search Inputs on both the Text and Shapes pane based on the `showTextAndShapesSearchInput` feature flag.

## Relevant Technical Choices

In both the `TextPane` and `ShapePane` components the following jsx is conditionally rendered based on the feature flag:

```jsx
      {showTextAndShapesSearchInput && (
        <SearchInput
          value={''}
          placeholder={__('Search', 'web-stories')}
          onChange={() => {}}
          disabled
        />
      )}
```

## User-facing changes
The Search Input on both the Text and Shape panes will be hidden from users if the flag is set to false.


## Testing Instructions

When launching into the workspace editor, the Search Input on both the Text and Shape panes should not be visible by default. Flipping the`showTextAndShapesSearchInput` flag in `includes/Story_Post_Type.php` to true and refreshing should make it visible on both

![2098-search-input-flag](https://user-images.githubusercontent.com/2755722/83782459-d4c3ea00-a65d-11ea-9b50-29fa414355ae.gif)


---



#2098
